src/utils/daos_hdlr.c

```diff
@@ -1919,6 +1918,49 @@ cont_rollback_hdlr(struct cmd_args_s *ap)
 	return rc;
 }
 
+int
+cont_list_objs_hdlr(struct cmd_args_s *ap)
+{
+	static const int OID_ARR_SIZE	= 8;
+	daos_obj_id_t oids[OID_ARR_SIZE];
+	daos_handle_t oit;
+	daos_anchor_t anchor;
+	uint32_t oids_nr;
+	int rc, rc2 = 0, i;
```

Comment:
Should align the variable names with each other on tabs

Reply:
we are not super consistent on this point

Reply:
Yeah. It's pointed out in our project style guide, though.

Reply:
ok